spec/test-utils/mockEndpoints.ts

```diff
@@ -56,3 +58,35 @@ export function mockSetupCrossSigningRequests(): void {
         {},
     );
 }
+
+/**
+ * Mock out requests to  `/room_keys/version`.
+ *
+ * Returns `404 M_NOT_FOUND` for GET requests until `POST room_keys/version` is called.
+ * Once the POST is done, `GET /room_keys/version` will return the posted backup
+ * instead of 404.
+ *
+ * @param backupVersion - The version of the backup to create
```

Comment:
this still needs an update, per https://github.com/matrix-org/matrix-js-sdk/pull/3662#discussion_r1310361726